src/opcodes.c

```diff
@@ -55,6 +55,25 @@ void writeOnMemoryOrAccumulator(unsigned int address, unsigned char result, stru
     : writeBus(address, result, bus);
 }
 
+void sumOperation(signed int value, struct CPU *cpu, struct BUS *bus) {
+  signed int sum = cpu->accumulator + value + cpu->status.carry;
+  cpu->status.carry = sum > 0xFF;
+  cpu->status.overflow = ~(cpu->accumulator ^ value) & (cpu->accumulator ^ sum) & 0x80;
+  cpu->accumulator = sum;
+  cpu->status.negative = (cpu->accumulator >> 7);
+  cpu->status.zero = cpu->accumulator == 0;
+}
+
+void adcOpcode(unsigned int address, struct CPU *cpu, struct BUS *bus) {
+  signed int valueFromAddress = readBus(address, bus);
+  sumOperation(valueFromAddress, cpu, bus);
+}
+
+void sbcOpcode(unsigned int address, struct CPU *cpu, struct BUS *bus) {
```

Comment:
could you put this function in alphabetic order?

Reply:
Done!